lib/open_api_spex/plug/app_env_cache.ex

```diff
@@ -1,4 +1,6 @@
 defmodule OpenApiSpex.Plug.AppEnvCache do
+  @moduledoc false
```

Comment:
Not sure we should hide this module from the docs. Users on older OTP versions will need to configure their cache to use it.